doc/whats_new.rst

```diff
@@ -23,6 +23,8 @@ Changelog
 
     - Add label center of mass function :func:`mne.Label.center_of_mass` by `Eric Larson`_
 
+    - Added :func:`mne.viz.ica.plot_properties` that allows ploting of independent component properties similar to ``pop_prop`` in EEGLAB. Also :class:`mne.preprocessing.ica.ICA` has ``plot_properties`` method now. Added by `Mikołaj Magnuski`_
```

Comment:
use as ref :func:`:class:`mne.preprocessing.ica.ICA.plot_properties`

:func:`mne.viz.ica.plot_properties` and should be documented as :func:`mne.viz.plot_ica_properties` so name should be plot_ica_properties. See what we do for Epochs, Evoked etc.